docs/faq.md

```diff
@@ -0,0 +1,23 @@
+# FAQ
+
+## When is it appropriate to use Ark instead of etcd's built in backup/restore?
+
+If you only want to backup/restore a single etcd cluster, you may be better of using etcd's backup
+and restore tooling. However, doing so restricts you to reconstructing your Kubernetes cluster data
+exactly as it was preserved. Etcd's restore tooling is good if what you want is to recover from
+data loss in a single etcd cluster, but does not support more sophisticated restores such as cluster
+migrations and restoring Kubernetes state stored across multiple etcd clusters.
+
+Ark is useful for:
+
+* backing up both Kubernetes resources and persistent volume state
+* cluster migrations
+* backing up a subset of your Kubernetes resources
+* backing up Kubernetes resources that are stored across multiple etcd clusters (for example if you
+  run a custom apiserver)
+
+## Will Ark restore my Kubernetes resources exactly the way they were before?
+
+Yes, with some exceptions. For example, when Ark restores pods it deletes the `nodeName` from the
```

Comment:
@ncdc should we also mention the volume mount removal for the default service account token?

e.g. if we're mentioning "exceptions" we should be able to say something along the lines of:

> The chief differences that Ark makes during a restore are:
> * ...
> * ...

So that readers can feel like they got most of what they needed to know without resorting to skimming the source code. (If we mention exceptions and only describe one, they may feel less comfortable).

Reply:
Also, I'm wondering if this particular answer should actually be documented in https://github.com/julia-stripe/ark/blob/bbef7a3e33f0d8d9ad5e948c09f557ec616bb11a/docs/concepts.md#3-restores, or if that is too deep in the docs that no one will see it.

Reply:
i'm going to leave this one to you all to resolve because I don't actually know what the chief differences with Ark are.

Reply:
I think it is a good idea to document the behavior of each custom backup action and restorer. I don't think it needs to be in this PR. For now it's ok to link to the pod restorer source code. Longer term, that link should be replaced to point at the docs on the actions and restorers.

Reply:
SGTM!